drivers/net/ksz9477_reg.h

```diff
@@ -118,6 +118,18 @@
 #define SGMII_AUTONEG_CONTROL_TC_MASTER    (1 << 3)
 #define SGMII_AUTONEG_CONTROL_LINK_STATUS  (1 << 4)
 
+/* Port Mirroring Control Register */
+
+#define KSZ9477_PORT_MIRROR_CONTROL(p)     KSZ9477_PORT_REG(p, 0x800)
+#define PORT_MIRROR_SNIFFER_PORT           (1 << 1)
```

Comment:
Should there be KSZ9477_ prefix for these bit field defines also as they are anyway chip specific fields?

Reply:
Done